src/offset_of.rs

```diff
@@ -95,6 +95,14 @@ macro_rules! offset_of {
         // Compute offset.
         _memoffset_offset_from!(field_ptr, base_ptr)
     }};
+    ($parent:ty, $field:tt) => {{
```

Comment:
Please add a comment explaining why we have both of these branches in the macro.

Reply:
Also, `raw_field_tuple` looks a lot like `raw_field`... can't you move this branching all the way down to the field check?

Reply:
Maybe I'm not good with macro-by-example, but the branching seems to break down in the context of another macro-by-example - it won't consider both branches. I can try playing with this some more to show you what I mean.

Reply:
In fact, I may even have to export an `offset_of_tuple` so that I can use this from my macro.

Reply:
Oh, this is the top-level macro... yeah looks like once a token is in some category it will not get re-evaluated further down, that kind of makes sense.

Probably best indeed to make it a separate macro then.